config/locales/en.yml

```diff
@@ -90,47 +90,47 @@ en:
-              blank: "^You need to pick a level"
+              blank: "^Select a course level"
             qualification:
-              blank: "^You need to pick an outcome"
+              blank: "^Select an outcome"
             maths:
-              inclusion: "^Pick an option for Maths"
+              inclusion: "^Select an option for maths"
             english:
-              inclusion: "^Pick an option for English"
+              inclusion: "^Select an option for English"
             science:
-              inclusion: "^Pick an option for Science"
+              inclusion: "^Select an option for science"
             enrichments:
               blank: "^Complete your course information before publishing"
             sites:
-              blank: "^You must pick at least one location for this course"
-              site_urn_not_publishable: "^You must provide a Unique Reference Number (URN) for all course locations"
+              blank: "^Select at least one location for this course"
+              site_urn_not_publishable: "^Enter a Unique Reference Number (URN) for all course locations"
             age_range_in_years:
-              blank: "^You need to pick an age range"
+              blank: "^Select an age range"
             program_type:
-              blank: "^You need to pick an option"
+              blank: "^Select a program type"
             subjects:
               blank: "^There is a problem with this course. Contact support to fix it (Error: S)"
-              course_creation: "^You must pick at least one subject"
-              duplicate: "^You've already selected this subject - you can only select a subject once"
+              course_creation: "^Select at least one subject"
+              duplicate: "^You have already selected this subject. You can only select a subject once"
             modern_languages_subjects:
-              select_a_language: "^You must pick at least one language"
+              select_a_language: "^Select at least one language"
             study_mode:
-              blank: "^You need to pick an option"
+              blank: "^Select a study mode"
             applications_open_from:
-              blank: "^You must say when applications open from"
+              blank: "^Select when applications will open and enter the date if applicable"
             base:
               duplicate: "This course already exists. You should add further locations for this course to the existing profile in Publish"
-              visa_sponsorship_not_publishable: "You must say whether you can sponsor visas"
-              provider_ukprn_not_publishable: "You must provide a UK provider reference number (UKPRN)"
-              provider_ukprn_and_urn_not_publishable: "You must provide a UK provider reference number (UKPRN) and URN"
+              visa_sponsorship_not_publishable: "Select if you can sponsor visas"
+              provider_ukprn_not_publishable: "Enter a UK Provider Reference Number (UKPRN)"
```

Comment:
please don't merge this yet

Reply:
👍 Still trying to get the tests to pass…